content/docs/5.1/concepts/explore-tikv-features/cas.md

```diff
@@ -0,0 +1,119 @@
+---
+title: CAS on RawKV
+description: Compare And Swap
+menu:
+    "5.1":
+        parent: Features
+        weight: 4
+---
+
+This page walks you through a simple demonstration of how to use TiKV's `compare-and-swap`.
+
+In RawKV, `compare-and-swap (CAS)` is an atomic instruction used in multi-threading to achieve synchronization. It compares the `current_value` of a `key` with a given value and, only if they are the same, modifies the contents of that `key` to a new given value. This is done as a single atomic operation. The atomicity guarantees that the new value is calculated based on up-to-date information; if the value had been updated by another thread in the meantime, the write would fail.
+
+## Prerequisites
```

Comment:
I don't see anywhere below we start a TiKV cluster by TiUP. And I think `cargo run` requires an existing cluster.

Reply:
thx, i forget that.

Reply:
This is a section to describe how to set up the tikv cluster in the before, you only need to reference that document, no need to write the same content again.